src/Common/src/System/Globalization/IdnMapping.cs

```diff
@@ -131,7 +135,13 @@ public string GetUnicode(string ascii, int index, int count)
             // We're only using part of the string
             ascii = ascii.Substring(index, count);
```

Comment:
Why is this not getting the same treatment as the previous one?  Seems like we're paying the cost of using unsafe without getting the benefit.